Hush/Assets/Scripts/Weapon/PlayerSword.cs

```diff
@@ -79,12 +93,22 @@ public float AttemptCrit(float damage)
             return damage;
         }
 
+        private void LightAttack()
+        {
+            playerAnimator.SetTrigger(PlayerAnimator.LightAttack);
+        }
+
+        private void HeavyAttack()
+        {
+            playerAnimator.SetTrigger(PlayerAnimator.HeavyAttack);
+        }
+
         private void OnTriggerEnter(Collider col)
         {
             if (!col.CompareTag(Tags.Enemy) && !col.CompareTag(Tags.Dome)) 
                 return;
         
-            var stateInfo = playerAnimator.GetCurrentAnimatorStateInfo(PlayerAnimator.Layer.UpperBody);
+            var stateInfo = playerAnimator.GetCurrentAnimatorStateInfo((int)PlayerAnimator.Layer.UpperBody);
```

Comment:
setting the type of the Layer enum might save you the cast to int maybe, not sure exactly how it works in C#

Reply:
it is set! (See Hush/Assets/Scripts/Player/Enums/PlayerAnimator.cs) it doesn't auto cast still!